variables.tf

```diff
@@ -385,6 +385,12 @@ variable "control_plane_access_source" {
   type        = list(any)
 }
 
+variable "control_plane_nsgs" {
+  default     = []
+  description = "A list of the network security groups (NSGs) ids to apply to the cluster endpoint."
+  type        = list(any)
```

Comment:
any reason to make it as list(any) instead of list(string)

Reply:
I think that was applied by the linting tool. I'll make it explicitly string